.github/workflows/python_unittests.yaml

```diff
@@ -15,13 +15,17 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install python3 python3-pip netcdf-bin
-          sudo pip3 install pyyaml jinja2 f90nml
+          sudo pip3 install pyyaml jinja2==2.11 f90nml
           sudo pip3 install numpy matplotlib basemap
+          sudo ln -sf /bin/bash /bin/sh
```

Comment:
It is the sourcing bash config file from python that is problematic
```Python
    code = dedent(
        f"""      #!/bin/bash
      t1="./t1.{pid}"
      t2="./t2.{pid}"
      (set -o posix; set) > $t1
      {{ . {config_file}; set +x; }} &>/dev/null
      (set -o posix; set) > $t2
      diff $t1 $t2 | grep "> " | cut -c 3-
      rm -rf $t1 $t2
    """
    )   
    (_, config_str, _) = run_command(code)
```
I will specify the default shell instead of symlinking.